docs/tutorials/zq-primer.md

```diff
@@ -892,20 +892,20 @@ argument to run all the transformations in one fell swoop:
 zq -I transform.zed prs.json > prs.zng
 ```
 
-## Running Analyics
+## Running Analytics
 
 Now that we've cleaned up our data, we can reliably and easily run analytics
-on finalized ZNG file `prs.zng`.
+on the finalized ZNG file `prs.zng`.
 
 Zed gives us the best of both worlds of JSON and relational tables: we have
-the structure and clarity of the relaional model while retaining the flexibility
+the structure and clarity of the relational model while retaining the flexibility
 of JSON's document model.  No need to create tables then issue SQL insert commands
 to put your clean data into all the right places.
 
 Let's start with something simple.  How about we output a "PR Report" listing
-the title of each PR along with it's PR number and creation date:
+the title of each PR along with its PR number and creation date:
 ```mdtest-command dir=docs/tutorials
-zq -f table '{DATE:created_at,"NUMBER":"PR #${number}",TITLE:title}' prs.zng
+zq -f table '{DATE:created_at,NUMBER:"PR #${number}",TITLE:title}' prs.zng
```

Comment:
At first I assumed the quotes around `"NUMBER"` were significant for some reason, but the surrounding text didn't say anything and removing them didn't change the results, so...